pkg/integration/tests/sync/fetch_prune.go

```diff
@@ -41,7 +41,7 @@ var FetchPrune = NewIntegrationTest(NewIntegrationTestArgs{
 		t.Views().Branches().
 			Lines(
 				Contains("master"),
-				Contains("branch_to_remove").Contains("upstream gone"),
+				Contains("branch_to_r…").Contains("upstream gone"),
```

Comment:
It doesn't feel good to make assumptions about the exact layout in tests. Any better suggestions?

Reply:
huh interesting. Is this because 'upstream gone' is part of the status and so it cuts into the available space? That is super annoying. Maybe we should just have a red x instead.

But the broader question remains: how do we deal with display things which are a result of a small window: one way is to just have a very large window in tests so that we don't need to worry about this stuff (at the moment we have a fairly tall window). But this means tests are more likely to fail locally if run within your own terminal window. What do you think?

Reply:
Argh, I didn't even realize that the fix above only fixed the test for headless mode, but makes it fail for running the test in a terminal. For now I decided to just make the view a bit wider in headless mode, this takes care of it; it was unrealistically narrow anyway. See 9e37ae3f5d39. It's still a bit of a hack, but I feel this is good enough for now. And I don't worry too much about developers with tiny monitors who get a failure when running this non-headless.

As for the 'upstream gone' part, I like that it is so prominent and would vote against changing it to a red x. For the record, it used to say just "↑gone" at one point, it was changed to "(upstream gone)" in April last year (dfb293c98599).

Reply:
I agree: being explicit with 'upstream gone' is good. Happy with your commit.